src/main/java/edu/greenblitz/robotName/utils/motors/GBTalonFXPro.java

```diff
@@ -22,4 +25,20 @@ public StatusCode setControl(ControlRequest control) {
 	public StatusCode applyConfiguration(TalonFXConfiguration configuration) {
 		return super.getConfigurator().apply(configuration);
 	}
+
+	@Override
+	public boolean isConnected() {
+		try {
+			getDeviceID();
+			return true;
+		}
+		catch (Exception e) {
+			return false;
+		}
```

Comment:
Return false at end

Reply:
java doesn't allow having try without catch or an unreachable return. could you specify what you would like me to write?